### DISCUSSION TITLE
PHP version mismatch

### DISCUSSION THREAD

**Discussion by amitpri** [laravel/framework#39019] [Q&A]
- Laravel Version: 8.6.2
- PHP Version: 7.3
- Database Driver & Version:

### Description:

I am trying to install any latest version of Laravel 8x through Forge. I get the following error which clearly says php 7.4 is needed. However all the laravel 8x documents say it needs 7.3 version of php. 
I tried to fix the problem by taking little older version of laraval 8 . I already have laravel ("laravel/framework": "^8.40",) running fine for a different website with php7.3. However the same codebase when built freshly(compsoer update) need the following packages which is not working with PHP7.3 anymore.

Problem 1
- Installation request for league/commonmark 2.0.2 -> satisfiable by league/commonmark[2.0.2].
- league/commonmark 2.0.2 requires php ^7.4 || ^8.0 -> your PHP version (7.3.0) does not satisfy that requirement.
Problem 2
- Installation request for league/config v1.1.1 -> satisfiable by league/config[v1.1.1].
- league/config v1.1.1 requires php ^7.4 || ^8.0 -> your PHP version (7.3.0) does not satisfy that requirement.
Problem 3
- league/commonmark 2.0.2 requires php ^7.4 || ^8.0 -> your PHP version (7.3.0) does not satisfy that requirement.
- laravel/framework v8.61.0 requires league/commonmark ^1.3|^2.0 -> satisfiable by league/commonmark[2.0.2].
- Installation request for laravel/framework v8.61.0 -> satisfiable by laravel/framework[v8.61.0].

**Comment by X-Coder264** [comment]
https://github.com/laravel/framework/issues/38747 ;)
